scripts/python/verify_agent_os.py

```diff
@@ -1,14 +1,26 @@
 # Script verifies the operating system for the platform on which it is being run
 # Used in build pipelines to verify the build agent os
-# Variable: The name of the os to verfy against
-
+# Variable: The friendly name or image name of the os to verfy against
+from __future__ import print_function
 import sys
 import platform
-os_parameter = sys.argv[1]
-print ("Parameter passed by matrix -", os_parameter)
-os_parameter = 'Darwin' if(os_parameter.lower() == 'MacOS'.lower()) else os_parameter
+
+os_parameter = sys.argv[1].lower()
+if os_parameter.startswith('macos'):
+	os_parameter = 'macOS'
+elif os_parameter.startswith('ubuntu'):
```

Comment:
How about if 'Linux' is passed as the os_parameter?

Reply:
In this case I don’t need to change anything. These if statements are here to help convert from a VM Image Name to a friendly name.